Gemfile

```diff
@@ -45,14 +45,14 @@ gem "hamlit",                           "~>2.11.0"
 gem "inifile",                          "~>3.0",             :require => false
 gem "inventory_refresh",                "~>2.1",             :require => false
 gem "kubeclient",                       "~>4.0",             :require => false # For scaling pods at runtime
-gem "linux_admin",                      "~>2.0", ">=2.0.1",  :require => false
+gem "linux_admin",                      "~>3.0",             :require => false
 gem "listen",                           "~>3.2",             :require => false
 gem "manageiq-api-client",              "~>0.3.6",           :require => false
 gem "manageiq-loggers",                 "~>1.0", ">=1.1.1",  :require => false
 gem "manageiq-messaging",               "~>1.0", ">=1.4.1",  :require => false
 gem "manageiq-password",                "~>1.0",             :require => false
 gem "manageiq-postgres_ha_admin",       "~>3.2",             :require => false
-gem "manageiq-ssh-util",                "~>0.1.1",           :require => false
+gem "manageiq-ssh-util",                "~>0.2.0",           :require => false
```

Comment:
NOTE I kept this as `"~>major.minor.patch"` because that is what was there before, but we could open this up to `"~>0.2"`

Reply:
We probably want to keep it locked down the way it is unless we want to release v1.0 then switch to `~>1.0`.

Reply:
I think it's stable enough for a 1.0, the last release is a couple days short of 4 years ago 😆